hypermapper/schema.json

```diff
@@ -237,7 +237,7 @@
       "optimization_method":{
         "type": "string",
         "description": "Method to use for the multi-objective optimization in Hypermapper.",
-        "enum": ["bayesian_optimization", "random_scalarizations", "local_search", "prior_guided_optimization", "evolutionary_optimization"],
+        "enum": ["bayesian_optimization", "bayesian_optimization_c_backend", "random_scalarizations", "local_search", "prior_guided_optimization", "evolutionary_optimization"],
```

Comment:
I think it would be better if we had a separate option for the C backend, rather than a separate optimizer. E.g. a new boolean field like "use_c_backend".

Reply:
Sure thing, I'll update that in the next revision